lab_3_generate_by_ngrams/main.py

```diff
@@ -208,6 +299,19 @@ def build(self) -> int:
         In case of corrupt input arguments or methods used return None,
         1 is returned
         """
+        if not isinstance(self._encoded_corpus, tuple) or \
+                len(self._encoded_corpus) == 0:
+            return 1
+        n_grams = self._extract_n_grams(self._encoded_corpus)
+        if not isinstance(n_grams, tuple) or \
```

Comment:
If it not a tuple, then it is `None`, according to annotation, and `None` should be checked with `is` operator

Reply:
Or, to make it ever simpler, both empty containers and `None` map to `False` in python

Reply:
Fixed